doc/build/core/connections.rst

```diff
@@ -534,7 +534,7 @@ Enabling ``threadlocal`` is achieved as follows::
 
 The above :class:`.Engine` will now acquire a :class:`.Connection` using
 connection resources derived from a thread-local variable whenever
```

Comment:
ah.  actually, this whole section should have been removed as part of https://gerrit.sqlalchemy.org/#/c/sqlalchemy/sqlalchemy/+/1356/.   there's no more "threadlocal" :) .    you can alter this PR for that instead :)